fund-aggregator/main.go

```diff
@@ -0,0 +1,147 @@
+package main
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/genaccounts"
+	"github.com/cosmos/cosmos-sdk/x/staking"
+	"github.com/kelseyhightower/envconfig"
+	"github.com/medibloc/panacea-core/app"
+	"github.com/medibloc/panacea-core/types/util"
+	tmtypes "github.com/tendermint/tendermint/types"
+	"log"
+	"os"
+)
+
+type Config struct {
+	GenesisPath string `envconfig:"GENESIS_PATH" required:"true"`
+}
+
+func main() {
+	sdkConfig := sdk.GetConfig()
+	sdkConfig.SetCoinType(371)
+	sdkConfig.SetFullFundraiserPath("44'/371'/0'/0/0")
+	sdkConfig.SetBech32PrefixForAccount(util.Bech32PrefixAccAddr, util.Bech32PrefixAccPub)
+	sdkConfig.SetBech32PrefixForValidator(util.Bech32PrefixValAddr, util.Bech32PrefixValPub)
+	sdkConfig.SetBech32PrefixForConsensusNode(util.Bech32PrefixConsAddr, util.Bech32PrefixConsPub)
+	sdkConfig.Seal()
+
+	var config Config
+	if err := envconfig.Process("", &config); err != nil {
+		log.Panic(err)
```

Comment:
`log.Panic(...)` is write log and panic occur?

Reply:
yeah :) it's quite tricky and it's considered that it's not good to use it. I just used it, so that we can see the error clearly. But, in production codes, it would be better to just print logs and `exit(1)`.